src/DataFixtures/ORM/LoadReferentManagedUserData.php

```diff
@@ -25,7 +25,7 @@ public function load(ObjectManager $manager)
             'original_id' => $this->getReference('adherent-1')->getId(),
             'email' => $this->getReference('adherent-1')->getEmailAddress(),
             'postal_code' => $this->getReference('adherent-1')->getPostalCode(),
-            'city' => $this->getReference('adherent-1')->getCity(),
+            'city' => $this->getReference('adherent-1')->getCityName(),
```

Comment:
I can say yes, just this method fills the tests datas and the data of this field was not really used in tests before.